client/core/trade.go

```diff
@@ -340,6 +340,7 @@ func (t *trackedTrade) negotiate(msgMatches []*msgjson.Match) error {
 		fillRatio := float64(trade.Filled()) / float64(trade.Quantity)
 		details := fmt.Sprintf("%s order on %s-%s %.1f%% filled (%s)",
 			strings.Title(sellString(trade.Sell)), unbip(t.Base()), unbip(t.Quote()), fillRatio*100, t.token())
+		log.Debugf("trade order %v matched with %d orders", t.ID(), len(msgMatches))
```

Comment:
May prove to be overly redundant or not valuable, but I figure it would be good to know how many matches were made for this order.